assets/base.css

```diff
@@ -730,19 +730,37 @@ details > * {
 }
 
 .underlined-link,
-.customer a {
+.customer a,
+.inline-richtext a {
   color: rgba(var(--color-link), var(--alpha-link));
   text-underline-offset: 0.3rem;
   text-decoration-thickness: 0.1rem;
   transition: text-decoration-thickness ease 100ms;
 }
 
 .underlined-link:hover,
-.customer a:hover {
+.customer a:hover,
+.inline-richtext a:hover {
   color: rgb(var(--color-link));
   text-decoration-thickness: 0.2rem;
 }
 
+.inline-richtext a,
+.rte.inline-richtext a {
+  color: currentColor;
+}
+
+@media screen and (min-width: 750px) {
+  .banner--desktop-transparent .rte a {
+    color: currentColor;
+  }
+
+  .banner--desktop-transparent .inline-richtext a:hover,
+  .banner--desktop-transparent .rte a:hover {
+    color: currentColor;
+  }
+}
```

Comment:
That `banner--desktop-transparent` class (and these rules) are actually used for the Image Banner _and_ the Slideshow section when "Show container on desktop" is unchecked. I opted to include the rule once in `base.css` instead of duplicating these styles in both `section-image-banner.css` and `component-slideshow.css`.

Reply:
Ah I see. The slideshow section is also using/loading `section-image-banner.css` at the moment. So I think it might still be a good idea to move the stylings to that file only 👍

Reply:
Ah, good call, I missed that. Updated in d956d6f.